### DISCUSSION TITLE
Windows on ARM64 PreBuilds?

### DISCUSSION THREAD

**Discussion by jorbig** [FreeRDP/FreeRDP#9869] [Ideas]
With the rise of Windows on Arm devices (eg. the upcoming Snapdragon X Elite devices), which are very convenient as thin RDP clients because of their battery life and fanless options, isn't it time for the Jenkins server to serve ARM64 builds next to X86? At least it could replace the somewhat deprecated 32-bit versions.

**Comment by akallabeth** [comment]
1. we don´t currently have that set up (but if there is someone willing to provide such a build node we can integrate it)
1. we don´t actually provide pre-builds, but `nightly builds`  for testing purposes only.  (just check the memory consumption and check the awfully slow execution due to `asan` and `WINPR_ASSERT` being verbose)

**Comment by jorbig** [comment]
It's just that I (and I guess I'm not the only one) lack the skill of compiling... even with the instructions in the Wiki. For example, when I'm told to build OpenSSL, I have no clue what my install path is, let alone what libdir I have to choose (lib or lib64). I feel so stupid, but it would be really nice if someone shared a default build. Because even the X64 nightly builds perform way better than Microsoft's built-in RDP client.
